### PR TITLE
Make default EmptyMessage value a non-null object

### PR DESCRIPTION
**Pull request by andreamlin**
This goes along with https://github.com/googleapis/gax-java/pull/685, which allows EmptyMessage to be constructed like a proto message.

This allows EmptyMessage to have a non-null test value, just like com.google.protobuf.Empty is constructed in https://github.com/googleapis/gapic-generator/blob/12e27eda15d2823684bba0a070252016bea9334e/src/test/java/com/google/api/codegen/gapic/testdata/java/java_library.baseline#L8368

### CODE REVIEW

File: src/main/java/com/google/api/codegen/transformer/java/JavaSchemaTypeNameConverter.java [hunk 1]
```diff
@@ -51,9 +51,6 @@ public JavaSchemaTypeNameConverter(String implicitPackageName, JavaNameFormatter
   }
 
   private static String getPrimitiveTypeName(Schema schema) {
-    if (schema == null) {
```
Comment (andreamlin) [inline src/main/java/com/google/api/codegen/transformer/java/JavaSchemaTypeNameConverter.java]: this was never called (also not called in production, when generating the compute API)

File: src/main/java/com/google/api/codegen/transformer/java/JavaSchemaTypeNameConverter.java [hunk 2]
```diff
@@ -94,9 +91,6 @@ private static String getPrimitiveZeroValue(Schema schema) {
     if (primitiveType.equals("java.lang.String")) {
       return "\"\"";
     }
-    if (primitiveType.equals("com.google.api.gax.httpjson.EmptyMessage")) {
```
Comment (andreamlin) [inline src/main/java/com/google/api/codegen/transformer/java/JavaSchemaTypeNameConverter.java]: this was never called (also not called in production, when generating the compute API)

**Comment by andreamlin** [comment]
Note: this will be checked into a different branch (origin_lro) that isn't master. The compute-discovery-test will fail because gax-java hasn't yet been released with the proposed changes.

**Comment by hzyi-google** [comment]
Please do remember to release gax.

**Pull request merged**